nulib/meadow
# Update pipeline lambdas to AWS SDK v3 for nodejs18.x runtime compatibility

# Summary 
Update pipeline lambdas to AWS SDK v3 for nodejs18.x runtime compatibility

Terraform already applied to staging.

# Specific Changes in this PR

- Update all pipeline lambdas to use v3 of the AWS SDK for JavaScript

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test

Run some filesets of different types and roles through. Make sure everything comes out alright.

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
